### ISSUE TITLE
sendMessage, need stringify keyboard markup param

### ISSUE THREAD

**Issue by osv** [Suppen/Telegram-API-wrapper-for-JS#1]
Hello, nice work. But one remark,

Looking in doc of your js:

``` js
    ReplyKeyboardMarkup: function() {
        /** ReplyKeyboardMarkup's keyboard, Array of Array of String **/
        this.keyboard = [];
```

Must to say, telegram expect json stringified param, so I do:

``` js
api.sendMessage(chat_id, 'hello',
                          true /* disable_web_page_preview*/,
                          0, /* reply_to_message_id*/
                          JSON.stringify({ /*  reply_markup */
                            keyboard: [
                              ['yes', 'no'],
                              [ 'cancel'],
                            ],
                            resize_keyboard: true
                          }));
```

You may test it:

```
https://api.telegram.org/bot9999999:AAAAAAAAAAAAAAAAAAAA/sendMessage?chat_id=99999999&text=hello&reply_markup={%22keyboard%22:[[%22option1%22],[%22option2%22]]}
```

I create this issue because I plan to use this package and maybe you have plan to stringify in `_doRequest` (maybe need stringify all objects? Not sure for now) that will brake my api usage :)

See also: http://www.reddit.com/r/Telegram/comments/3bblz0/sendmessage_with_replykeyboardmarkup_problem

**Comment by Suppen** [comment]
Hello

I haven't used that part of the API myself yet, so I haven't noticed this. It does, however, make perfect sense that arrays and objects need to be serialized. Will fix once I get home, in about 8 hrs. My incompetent ISP has somehow fucked up my Internet, so I don't have remote access today...

**Comment by Suppen** [comment]
This should now be fixed in 0.9.0. Please tell me if something is still wrong
